src/Umbraco.Web.UI.Client/src/views/common/overlays/mediaPicker/mediapicker.controller.js

```diff
@@ -133,8 +133,7 @@ angular.module("umbraco")
                     });
                 $scope.currentFolder = folder;
 
-                // for some reason i cannot set cookies with cookieStore
-                document.cookie = "umbLastOpenedMediaNodeId=" + folder.id;
+                localStorageService.set("umbLastOpenedMediaNodeId", folder.id);

```

Comment:
I can't seem to see anything even get stored/saved into LocalStorage @perploug 
How best to reproduce? Pick a media start node (aka the Media PreValue picker) for a datatype & then I should see a localstorage value in my Chrome DevTools yes? If so then its not working for me or I am doing something very dumb.